atmos_dyamond_summer/create_artifact.jl

```diff
@@ -1,8 +1,276 @@
+        asn[:, :] = ncin["asn"][lonidx, latidx, 1]
+
+        # src (skin reservoir content)
+        src = defVar(ncout, "src", FT, ("lon", "lat",), attrib = ncin["src"].attrib)
+        src[:, :] = ncin["src"][lonidx, latidx, 1]
+
+        # ci (Sec-ice cover)
+        ci = defVar(ncout, "ci", FT, ("lon", "lat",), attrib = ncin["ci"].attrib)
+        ci[:, :] = ncin["ci"][lonidx, latidx, 1]
+
+        # swvl1 (volumetric soil water layer 1)
+        swvl1 = defVar(ncout, "swvl1", FT, ("lon", "lat",), attrib = ncin["swvl1"].attrib)
+        swvl1[:, :] = ncin["swvl1"][lonidx, latidx, 1]
+
+        # swvl2 (volumetric soil water layer 2)
+        swvl2 = defVar(ncout, "swvl2", FT, ("lon", "lat",), attrib = ncin["swvl2"].attrib)
+        swvl2[:, :] = ncin["swvl2"][lonidx, latidx, 1]
+
+        # swvl3 (volumetric soil water layer 3)
+        swvl3 = defVar(ncout, "swvl3", FT, ("lon", "lat",), attrib = ncin["swvl3"].attrib)
+        swvl3[:, :] = ncin["swvl3"][lonidx, latidx, 1]
+
+        # swvl4 (volumetric soil water layer 4)
+        swvl4 = defVar(ncout, "swvl4", FT, ("lon", "lat",), attrib = ncin["swvl4"].attrib)
+        swvl4[:, :] = ncin["swvl4"][lonidx, latidx, 1]
+
+        # slt (soil type)
+        slt = defVar(ncout, "slt", FT, ("lon", "lat",), attrib = ncin["slt"].attrib)
+        slt[:, :] = ncin["slt"][lonidx, latidx, 1]
+
+        # lsm (land-sea mask)
+        lsm = defVar(ncout, "lsm", FT, ("lon", "lat",), attrib = ncin["lsm"].attrib)
+        lsm[:, :] = ncin["lsm"][lonidx, latidx, 1]
+
+        # sr (surface roughness length)
+        sr = defVar(ncout, "sr", FT, ("lon", "lat",), attrib = ncin["sr"].attrib)
+        sr[:, :] = ncin["sr"][lonidx, latidx, 1]
+
+        # cvl (low vegetation cover)
+        cvl = defVar(ncout, "cvl", FT, ("lon", "lat",), attrib = ncin["cvl"].attrib)
+        cvl[:, :] = ncin["cvl"][lonidx, latidx, 1]
+
+        # cvh (high vegetation cover)
+        cvh = defVar(ncout, "cvh", FT, ("lon", "lat",), attrib = ncin["cvh"].attrib)
+        cvh[:, :] = ncin["cvh"][lonidx, latidx, 1]
+
+        # sdor (standard deviation or orography)
+        sdor = defVar(ncout, "sdor", FT, ("lon", "lat",), attrib = ncin["sdor"].attrib)
+        sdor[:, :] = ncin["sdor"][lonidx, latidx, 1]
+
+        # isor (anisotropy of sub-grid scale orography)
+        isor = defVar(ncout, "isor", FT, ("lon", "lat",), attrib = ncin["isor"].attrib)
+        isor[:, :] = ncin["isor"][lonidx, latidx, 1]
+
+        # anor (angle of sub-grid scale orography)
+        anor = defVar(ncout, "anor", FT, ("lon", "lat",), attrib = ncin["anor"].attrib)
+        anor[:, :] = ncin["anor"][lonidx, latidx, 1]
+
+        # slor (slope of sub-grid scale orography)
+        slor = defVar(ncout, "slor", FT, ("lon", "lat",), attrib = ncin["slor"].attrib)
+        slor[:, :] = ncin["slor"][lonidx, latidx, 1]
+    end
+    close(ncout)
+    return nothing
+end
+println("creating initial conditions for 0.98⁰ resolution")
```

Comment:
- [ ] Can you please add the description of the "low" resolution and a list of available variables to a README file ? 

- [ ] `small` kwarg appears unused. 

- [ ] `ClimaParams` is unused (are there other such unused modules / variables?

Reply:
`ClimaParams` is needed by `Thermodynamics.jl`
Description updated in README.md
List of variables is too big to mention individually. It is not clear if we will need all of these. But they have been added just be conservative.

Reply:
Indeed, thank you - I had missed that earlier ; I see that it's called in `TD.<fn>` instances.